### ISSUE TITLE
Pre-Requisite for `Create a network for the mun` lacks enough information

### ISSUE THREAD

**Issue by Emilgardis** [jrossignol/ContractPack-RemoteTech#20]
EDIT:: Issue has been edited as there was no real issue with the contract, only that it wasn't providing enough information.

~~Every pre-requisite is met except~~

> ~~Must have a RemoteTech constellation orbiting Kerbin with an active vessel dish (or omni antenna) with a range of at least 12,000 km]~~

~~AVC info:~~
~~https://hastebin.com/raw/asoluhugoq~~

~~I have two constellations around kerbin.~~
* ~~4 relay configuration in LKO with short range~~
* ~~3 relay configuration in KEO with Commtech 1 currently pointing at Gilly.~~

~~On the mun I have two satellites and two landers.~~

~~Is there any more information I could give?~~

### **Update::**
This contract's pre-requisite for the `Active vessel dish` should be reworded to something along the lines of

> Must have a RemoteTech constellation orbiting Kerbin with a dish set to active vessel (or omni antenna) with a range of at least 12,000 km

![Image](https://puu.sh/uaJym/fe59f12590.png)

**Comment by Emilgardis** [comment]
Found the solution, set your all your dishes to target `Active Vessel`.

The contract should probably specify this better, `with an active vessel dish` hinted at it, but I didn't get it until just now.

Changing issue to reflect this instead.

**Comment by AncAinu** [comment]
I agree with that ticket, it was very confusing, I had a dish set to "Mun" and took me a while to see that actually you can point a dish to "active vessel" which is the player controlled vessel at any moment.
